data/nsides/offsides/meta.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name: offsides
+description: OffSIDES is a database of individual drug side effect signals mined from the FDA's Adverse Event Reporting System. The innovation of OffSIDES
+    is that a propensity score matching (PSM) model is used to identify control drugs and produce better PRR estimates. In OffSIDES we focus on drug safety
+    signals that are not already established by being listed on the structured product label -- hence they are off-label drug side effects.
+targets:
+    - id: PRR
+      description: Proportional reporting ratio)
+      type: continuous
+      names:
+          - Proportional reporting ratio
+    - id: PRR_error
+      description: Standard error of the PRR estimate
+      type: continuous
+      names:
+          - Proportional reporting ratio error
+    - id: mean_reporting_frequency
+      description: Proportion of reports for the drug that report the side effect
+      type: continuous
+      names:
+          - mean reporting frequency
+identifier:
+    - id: drug_concept_name
+      description: RxNorm name string for the drug
+      type: categorical
+    - id: condition_concept_name
+      description: MedDRA identifier for the side effect
+      type: categorical
```

Comment:
Do we need both of them simultaneously for the ratio to be meaningful?
That is, a correct prompt would ask the model something like

"What is the proportional reporting ratio for <condition> for <drug>"

Reply:
yeah, so the prompt would be "what is the PRR of <condition_concept_name> for the <drug_concept_name>?". higher PRR means higher reported side effect for that particular drug.